_posts/2022-10-04-clicky-reflection-alm7468.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This file should be in the `alm7468` subdirectory. So I believe the overall filename would be:

~~~
_posts/alm7468/2022-10-04-clicky-reflection-alm7468.md
~~~

Reply:
I added the subfolder, thank you!